aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -31,14 +31,21 @@ module aptos_framework::aptos_governance {
     use aptos_framework::timestamp;
     use aptos_framework::voting;
 
-    /// Error codes.
+    /// The specified stake pool does not have sufficient stake to create a proposal
     const EINSUFFICIENT_PROPOSER_STAKE: u64 = 1;
+    /// This account is not the designated voter of the specified stake pool
     const ENOT_DELEGATED_VOTER: u64 = 2;
+    /// The specified stake pool does not have a sufficient remaining lockup to create a proposal or vote
     const EINSUFFICIENT_STAKE_LOCKUP: u64 = 3;
+    /// The specified stake pool has already been used to vote on the same proposal
     const EALREADY_VOTED: u64 = 4;
+    /// The specified stake pool must be part of the validator set
     const ENO_VOTING_POWER: u64 = 5;
+    /// Proposal cannot be resolved yet and thus the script hash cannot be added to the approved list
```

Comment:
This one is very weird... and I can't come up with a better sentence.

Proposal is not ready to be resolved.  Waiting on time or votes? (don't know how to do the second sentence)

I don't think it really matters about talking about the approved list